src/components/atom/Button.jsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
```

Comment:
I'm not sure this repo will ever have enough components that we need to start creating sub directories for things. Ditto for the icons directory.